spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RootController.java

```diff
@@ -134,6 +136,12 @@ public RootResource info() {
 		root.add(new Link(completionStreamTemplated).withRel("completions/stream"));
 		String completionTaskTemplated = entityLinks.linkFor(CompletionProposalsResource.class).withSelfRel().getHref() + ("/task{?start,detailLevel}");
 		root.add(new Link(completionTaskTemplated).withRel("completions/task"));
+
+		// Should this be in the isTasksEnabled() block above? Why are the completions endpoints not guarded by feature checks?
+		String taskTextToGraphTemplated = entityLinks.linkFor(TaskGraphResource.class).withSelfRel().getHref() + ("/parseTaskTextToGraph{?name,definition}");
```

Comment:
I have no idea what to do here. But this doesn't work. Tests fail when you mvn build it. The test failure message is unhelpful but if I run it anyway, it fails (curling to localhost:9393):

[{"logref":"IllegalArgumentException","message":"Cannot determine link for org.springframework.cloud.dataflow.rest.resource.TaskGraphResource! No EntityLinks instance found supporting the domain type!","links":[]}]

which is probably the big clue. I patterned what I did off what it did for completions/ endpoint - per my comment there I don't know why completions/task isn't guarded on being only active if the tasks feature is on? (Mine would be the same)